### ISSUE TITLE
1.4.4 pre-release Linux build not connecting

### ISSUE THREAD

**Issue by aplocher** [yakyak/yakyak#660]
The animated "Connecting..." screen just spins forever.  This is what I see in the console:

```
adam  BC-LINPC-02  ~  apps  yakyak-linux-x64  $  /home/adam/apps/yakyak-linux-x64/yakyak &
 adam  BC-LINPC-02  ~  apps  yakyak-linux-x64  $  hconnect
reconnecting 0
resolving proxy http://plus.google.com
resolving proxy https://plus.google.com
resolved proxy DIRECT
resolved proxy DIRECT
asking for login credentials
login: did-finish-load https://accounts.google.com/o/oauth2/programmatic_auth?hl=en&scope=https%3A%2F%2Fwww.google.com%2Faccounts%2FOAuthLogin+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.email&client_id=936475272427.apps.googleusercontent.com&access_type=offline&delegated_client_id=183697946088-m3jnlsqshjhh5lbvg05k46q1k4qqtrgn.apps.googleusercontent.com&top_level_cookie=1
login: programmatic auth
2017-04-17 4:39:51 WARN no SAPISID cookie
2017-04-17 4:39:51 WARN no SAPISID cookie
2017-04-17 4:39:51 WARN no SAPISID cookie
2017-04-17 4:39:51 WARN no SAPISID cookie
2017-04-17 4:39:51 ERROR getAuth failed { Error: socket hang up
    at TLSSocket.onHangUp (_tls_wrap.js:1117:19)
    at Object.onceWrapper (events.js:290:19)
    at emitNone (events.js:91:20)
    at TLSSocket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) code: 'ECONNRESET' }
connect_failed { Error: socket hang up
    at TLSSocket.onHangUp (_tls_wrap.js:1117:19)
    at Object.onceWrapper (events.js:290:19)
    at emitNone (events.js:91:20)
    at TLSSocket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) code: 'ECONNRESET' }
error connecting { Error: socket hang up
    at TLSSocket.onHangUp (_tls_wrap.js:1117:19)
    at Object.onceWrapper (events.js:290:19)
    at emitNone (events.js:91:20)
    at TLSSocket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) code: 'ECONNRESET' }
reconnecting 0
resolving proxy http://plus.google.com
resolving proxy https://plus.google.com
resolved proxy DIRECT
resolved proxy DIRECT
2017-04-17 4:39:55 INFO removing refresh token
connected 0
2017-04-17 4:39:55 INFO removing cookie store
asking for login credentials
login: did-finish-load https://accounts.google.com/o/oauth2/programmatic_auth?hl=en&scope=https%3A%2F%2Fwww.google.com%2Faccounts%2FOAuthLogin+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.email&client_id=936475272427.apps.googleusercontent.com&access_type=offline&delegated_client_id=183697946088-m3jnlsqshjhh5lbvg05k46q1k4qqtrgn.apps.googleusercontent.com&top_level_cookie=1
login: programmatic auth
2017-04-17 4:40:02 WARN no SAPISID cookie
2017-04-17 4:40:02 WARN no SAPISID cookie

```

**Comment by averissimo** [comment]
works fine for me.

Have you:

- Ever used hangouts before? there is a known bug if you don't have any contacts already (use web interface to add just one should solve this)
- Another known bug is that sometimes YakYak requires a mouse hover over the UI to connect (this is a stupid one that we cannot trace -- yet)
- Try to delete your ~/.config/yakyak directory (the only caveat is that you lose your configurations)

**Comment by aplocher** [comment]
Argh, seems to be working now.  I'll go ahead and close this.  

To answer your questions (although probably a moot point now):

- Yes, use YakYak on same machine, dual booted in Windows.  Also had an older build in this Linux instance although it had been months since I booted into it so it was a bit outdated.

- Don't think this was my issue, I'm pretty sure my self-diagnosed ADHD would have caused me to wiggle the cursor around sporadically and quite possibly throw a few clicks in there too.  Also probably alt+tabbed to it / away from it several times (in case activating/focusing on window means anything).

- If this happens again, will def try this...

It did happen every time I tried it (over the course of maybe an hour, tried it probably 8 or 9 times) so perhaps it was due to some outage or a routing issue with my ISP or something - who knows...

Thanks!
